front/pages/api/w/[wId]/vaults/[vId]/data_sources/managed.ts

```diff
@@ -51,6 +51,7 @@ export function getConnectorProviderCodec(): t.Mixed {
 const PostManagedDataSourceRequestBodySchema = t.type({
   provider: getConnectorProviderCodec(),
   connectionId: t.union([t.string, t.undefined]),
+  name: t.union([t.string, t.undefined]),
```

Comment:
Fixing this on this new route (was not used anywhere) -> we can indeed set a name for a managed ds, only for websites.